### ISSUE TITLE
Problem: comment ui is clumsy

### ISSUE THREAD

**Issue by porcellus** [EmurgoHK/cardanoupdate#573]
There are a few smaller issues:
- too much space between comments
- too much space around buttons
- too big buttons
- threads are hard to follow
We should aim for something like the comment ui of reddit:
![image](https://user-images.githubusercontent.com/1129990/48901523-039cb580-ee56-11e8-845f-8f0dc009d367.png)
We don't need voting or to copy all of it, but a few key point would be nice to implement:
- line starting from bottom of comments all the way next to subcomments
- buttons are much smaller and inside the comment area
- show/hide replies should be much less intrusive.

**Comment by akatrodiya** [comment]
@emurgobot claim
